libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -50,6 +53,14 @@ class LnurlPayInfoJob(
         try {
             request = Json.decodeFromString(LnurlInfoRequest.serializer(), payload)
             val nodeState = breezSDK.nodeInfo()
+            val ofpResp =
+                breezSDK.openChannelFee(OpenChannelFeeRequest(amountMsat = null))
+            val feeLimitMsats: ULong = config.autoChannelSetupFeeLimitMsats
+            val maxSendable = minOf(
+                nodeState.maxReceivableMsat,
+                feeLimitMsats / ofpResp.feeParams.proportional * 100u
```

Comment:
Is this correct? If

feeLimitMsats: 5000 msat
proportional: 4000
```
feeLimitMsats / ofpResp.feeParams.proportional * 100u
```
5000 / 4000 * 100 = 125 msat

Shouldn't we get 1250000 with a 5000 fee limit?

Reply:
Proportional unit is in parts per million, for example: 0.50

feeLimitMsats: 5_000_000 / 0.50 * 100
maxSendable that falls within fee limits: 1_000_000_000

In reverse: 1_000_000_000 * 0.50 / 100 = 5_000_000 (feeLimitMsats)

Reply:
```
sdk> open_channel_fee
{
  "fee_msat": null,
  "fee_params": {
    "min_msat": 2500000,
    "proportional": 4000,
    "valid_until": "2024-03-12T21:28:51.626Z",
    "max_idle_time": 4320,
    "max_client_to_self_delay": 432,
    "promise": "1f98ad865164d21005ee0828ac08f6300a7202c94b9712917486c372c444bf52353e0c06c8d86d61be465575295cbab63f89f7e266501194920253e02df4b87cf8"
  }
}
```

Reply:
Good catch.. should've checked it myself, don't know why I recalled proportional returning per
million value.

Corrected 5cfdc79415f2a58c4ab6ff91c1fef34240127d5f